packages/fxa-admin-server/src/gql/account/account.resolver.ts

```diff
@@ -373,4 +375,36 @@ export class AccountResolver {
       });
     return !!result;
   }
+
+  @Features(AdminPanelFeature.UnsubscribeFromMailingLists)
+  @Mutation((returns) => Boolean)
+  public async unsubscribeFromMailingLists(@Args('uid') uid: string) {
+    // Look up email. This end point is protected, but using a uid would makes it harder
+    // to abuse regardless.
+    const account = await this.db.account
```

Comment:
I think you would want to query the `emails` table for all emails associated with this uid and unsubscribe from them all.

Reply:
Interesting.  Do we use the primary email at the time of the subscription and then never update...?

Reply:
I considered this. I was talking to @clouserw about this. He indicated that they should be listening to events we broadcast about primary email changes, and therefore the primary should suffice.